src/components/magicui/text-animate.tsx

```
@@ -0,0 +1,407 @@
+  let segments: string[] = [];
+  switch (by) {
+    case "word":
+      segments = children.split(/(\s+)/);
+      break;
+    case "character":
+      segments = children.split("");
+      break;
+    case "line":
+      segments = children.split("\n");
+      break;
+    case "text":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)